core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -120,15 +129,23 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                       cy="70"
                       r="50"
                       ng-attr-fill="<[pathIconParameters[$index].thumbnailBgColor]>"/>
-              <image x="10"
+              <image ng-if="isCompletedExploration(node.getExplorationId())"
+                     x="0"
+                     y="20"
+                     height="100"
+                     width="100"
+                     ng-attr-xlink:href="<[getStaticImageUrl('/general/collection_paw.svg')]>"
+                     xlink:href=""></image>
+              <image ng-if="!isCompletedExploration(node.getExplorationId())"
+                     x="10"
                      y="30"
                      height="80"
                      width="80"
                      ng-attr-xlink:href="<[pathIconParameters[$index].thumbnailIconUrl]>"
                      xlink:href=""></image>
             </pattern>
           </defs>
-          <a xlink:href="<['/explore/' + node.getExplorationId()]>"
+          <a xlink:href="<['/explore/' + node.getExplorationId() + '?collection_id=' + collectionId]>"
```

Comment:
Can you refactor this into a function in the controller? Icky to do so much logic here.

Also, does this need an ng-href? Not sure what the right thing to do in svg is.